essay.tex

```diff
@@ -1,45 +1,97 @@
+Lean Development Governance is a governance method proposed to create a stable framework to allow Agile, Lean or similar software development methods to be implemented into a company or department.
+
+See this paper for details \cite{ambler2009applying}.
+
+\paragraph{Multiple Backlogs}\mbox{}\newline
+
+The Nokia Corporation took an interesting approach on one of their projects, utilising four different types of backlog to organise tasks across multiple teams.
+
+As shown in Figure \ref{fig:backlogs}, the system they developed consisted of two program level backlogs and two at team level. See this paper for details \cite{laanti2008implementing}.
+	
+\begin{figure}
+	\includegraphics[width=\linewidth]{"Nokia Multiple Backlogs".png}
+	\caption{Backlog Structure: Taken from \cite[p. ~1384]{laanti2008implementing}}
+	\label{fig:backlogs}
+\end{figure}
+
+
+\section{Analysis and Comparison}
+
+The widespread usage of Scrum of Scrums suggests that it becomes a crucial element of Agile Development as soon as team sizes start increasing as it facilitates constant communication without meetings becoming long and cumbersome.
+
+Lean Governance itself is not specifically tailored to improving scalability of Agile development but many of the principles of Lean Software Development overlap or mesh well with Agile. In addition to this, it has been suggested that many of the practices suggested by Lean Governance provide opportunity to employ Agile at a large scale. \cite{ambler2009scaling} One practice of Lean Governance that applies strongly to Agile development is Staged Program Delivery as it ensures constant delivery of working software, even if it lacks some features. 
+
+Nokia's approach was unique and provided a simple way to separate tasks between teams. However, while this method appears easy to set up, it is important to note that Nokia had many veteran program managers who were able to coordinate between teams and to help keep a long term goal in mind when planning sprints. Nokia also had problems with undisciplined practices, such as Scrum Masters acting as managers. This supports the idea that the effectiveness of Agile development is strongly affected by the presence of supportive management practices. \cite{maruping2009control} This reinforces the suggestion that a Lean Governance framework is crucial to providing the basis for Agile at Scale, as it promotes governance that supports the Agile values. Despite this, implementing Lean Governance is a major task that requires training management staff and reorganising company work-flow and therefore may not be suitable for all companies.  
```

Comment:
There is a hanging reference here. Should this be in the proceeding sentence, or the sentence it currently occupies? Typically, references to a source go at the end of a clause, often the end of the sentence.